.gitignore

```diff
@@ -0,0 +1,30 @@
+
```

Comment:
Can you please add Apache licence to all files. Thanx

Reply:
The Apache license has already been included in main.tf. I didn't include the license in files that aren't source code such as .gitignores and READMEs, see [https://github.com/GoogleCloudPlatform/opentelemetry-operations-js]. The JSON format also doesn't support comments as well.

Should a standalone LICENSE file be add to the 'monitoring' folder?